core/auth/auth.go

```diff
@@ -22,17 +22,18 @@ import (
 	"crypto/rsa"
 	"encoding/json"
 	"fmt"
+	"log"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/golang-jwt/jwt"
 	"github.com/google/uuid"
-	"github.com/rokwire/core-auth-library-go/authorization"
-	"github.com/rokwire/core-auth-library-go/authservice"
-	"github.com/rokwire/core-auth-library-go/authutils"
-	"github.com/rokwire/core-auth-library-go/sigauth"
-	"github.com/rokwire/core-auth-library-go/tokenauth"
+	"github.com/rokwire/core-auth-library-go/v2/authorization"
```

Comment:
With the upgrade to v2 of the auth library, we should update the service account system to use the `"all"` string instead of `nil`. Below are the two commits in #354 that make these changes. The second link shows the final state after the changes, and the first link shows some intermediate changes that may help make the second set of changes clearer.

- https://github.com/rokwire/core-building-block/pull/354/commits/396792b014d0a499b1aa5a32f2839a203bd3165c
- https://github.com/rokwire/core-building-block/pull/354/commits/0565ff975566841c8373b8cd71ecdd93983b9bbc